tasks/interactive-bots.md

```diff
@@ -1,12 +1,12 @@
 #GCI Tasks: interactive bots
 
-## Prerequisites
+## Project Essentials
 
 * A working Zulip development environment. See
   [here](https://github.com/zulip/zulip-gci/blob/master/README.md) for instructions on
   how to set one up.
 
-* You need to know how to create a GitHub pull request. Check out the
+* Knowledge on how to create a GitHub pull request. Check out the
```

Comment:
I don't see any reason why we should change it - the previous version is absolutely understandable